Aptoide/aptoide-client-v8
# [ASV-1426] Integrate Appnext

**What does this PR do?**
This PR aims at implementing appnext ads.  Since it is not one of the certified networks from MoPub, we had to implement a BaseAdapterConfiguration. 

**Database changed?**

   No

**Where should the reviewer start?**

- [ ] AppnextBaseAdapterConfiguration.java

**How should this be manually tested?**

Run the app and check if you have ads specially banners (home, search results, appview) and native ads (highlighted bundle in home, search results and appview similar apps bundle).
We will check how Appnext is working on the nightly 

**What are the relevant tickets?**

  Tickets related to this pull-request: [ASV-1426](https://aptoide.atlassian.net/browse/ASV-1426)

**Questions:**

   Does this add new dependencies which need to be added? (Eg. new libs, new keys, etc.) 




**Code Review Checklist**
